homework_13/task_1.py

```diff
@@ -0,0 +1,70 @@
+"""
+Описать класс Zoo.
+"""
+
+
+class Zoo:
+
+    penguin_family = ["Adam", "Jane"]
+
+    def __init__(self, address: str,
+                 area: int,
+                 visitors: int,
+                 animals: int,
+                 price: int
+                 ):
+        self._address = address
+        self._area = area
+        self._visitors = visitors
+        self._animals = animals
+        self._price = price
```

Comment:
```suggestion
        self._area =  int(area)
        self._visitors = int(visitors)
        self._animals = int(animals)
        self._price = int(price)
```
I'd recommend this to avoid setting a non-integer value

Reply:
Fixed.